fluentd/fluent.conf

```diff
@@ -135,18 +134,36 @@
     bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
     ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
   </filter>
+  <filter **kibana**>
+    type record_transformer
+    enable_ruby
+    <record>
+      log ${err || msg}
+    </record>
+    remove_keys req,res,msg,name,level,v,pid,err
+  </filter>
   <filter kubernetes.**>
     type flatten_hash
     separator _
   </filter>
+  <filter kubernetes.**>
+    type record_transformer
+    enable_ruby
+    <record>
+      hostname ${kubernetes_host}
+      message ${log}
+      @version 3.1
```

Comment:
This is part of the ECL initiative.  The idea for this is that if we make further fluentd config changes we would increment this version so that if there is an ES instance that persists multiple fluentd config versions we could use the @version tag so that we correlate data between versions (in the chance that there is no backwards compat).